### PR TITLE
New term: EPHB4-associated vascular malformation spectrum

### LINKED ISSUE #4941: EPHB4-Associated Vascular Malformation Spectrum [NTR/gene]
**Preferred gene-related syndrome label**
EPHB4-Associated Vascular Malformation Spectrum

**Synonyms**

**Parent term (use [OLS](https://www.ebi.ac.uk/ols/ontologies/mondo), or your favorite ontology browser)**
Vascular Malformation (MONDO:0024291)

**Definition**
Variants in the EPHB4 gene have been observed in individuals with capillary malformation-arteriovenous malformation (CM-AVM) (characterized by the presence of multiple small capillary malformations, mostly on the face and limbs, +/- other arteriovenous malformations or arteriovenous fistulas) as well as with lymphatic malformations, ranging in severity from severe non-immune hydrops fetalis to varicose veins and/or subclinical lymphatic anomalies, even within the same family (PMID: 27400125).  Evidence suggests that both of these presentations are caused by loss of function, though the exact mechanism by which this occurs is variable; some variants demonstrate reduced expression and defects in subcellular localization with aggregates, others have normal expression levels but reduced tyrosine kinase activity) (PMID: 33864021).  It has been hypothesized that the different presentations may be due to differences in forward vs. reverse signaling defects, but this remains to be elucidated.  

**Definition source (Please give PubMed ID, if applicable, in format PMID:#######)
PMID:2868770; PMID:30760892; PMID: 27400125; PMID:33864021

**Children terms (if applicable)** Should any existing terms be re-classified as children underneath this new proposed term? 
capillary malformation-arteriovenous malformation 2 (MONDO:0020785); lymphatic malformation 7 (MONDO:0015009)

**Your nano-attribution (ORCID) or URL for a working group**
If you don't have an ORCID, you can sign up for one [here](https://orcid.org/)
https://www.clinicalgenome.org/affiliation/40106/

### PR DESCRIPTION
**Pull request by sabrinatoro**
Fixes #4941

@nicolevasilevsky : do you think that this new term should follow https://github.com/monarch-initiative/mondo/blob/master/src/patterns/dosdp-patterns/disease_series_by_gene.yaml ?
If so, we should add the definition and the Equivalent axiom.
I think that "EPHB4-associated" (and the current definition) means "~due to variation in EPHB4", but I am not 100% certain. Thanks!

### CODE REVIEW

**Comment by nicolevasilevsky** [comment]
yes, I agree that this term should follow the pattern.

There appears to be conflicts on this branch but I'm unable to fix them for some reason. Would you mind making the changes and fixing the conflicts @sabrinatoro? thanks!

**Comment by sabrinatoro** [comment]
Resolving the conflicts created some weirdness : looking at the diff, it looks like the information for this new term was added to different terms in the ontology (!?). I am attempting to resolve this manually.

**Comment by sabrinatoro** [comment]
@nicolevasilevsky this is ready for you :-) Thank you

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -455486,6 +455489,15 @@ intersection_of: disease_has_feature NCIT:C68749 ! HER2/Neu Negative
...
+def: "Any vascular malformation in which the cause of the disease is a variation in the EPHB4 gene. Variants in the EPHB4 gene have been observed in individuals with capillary malformation-arteriovenous malformation (CM-AVM) (characterized by the presence of multiple small capillary malformations, mostly on the face and limbs, +/- other arteriovenous malformations or arteriovenous fistulas) as well as with lymphatic malformations, ranging in severity from severe non-immune hydrops fetalis to varicose veins and/or subclinical lymphatic anomalies, even within the same family (PMID: 27400125). Evidence suggests that both of these presentations are caused by loss of function, though the exact mechanism by which this occurs is variable; some variants demonstrate reduced expression and defects in subcellular localization with aggregates, others have normal expression levels but reduced tyrosine kinase activity) (PMID: 33864021). It has been hypothesized that the different presentations may be due to differences in forward vs. reverse signaling defects, but this remains to be elucidated." [MONDO:patterns/disease_series_by_gene, PMID:27400125, PMID:2868770, PMID:30760892, PMID:33864021]
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:455495]: @sabrinatoro for some reason I can't edit this branch myself. We shouldn't have the PMID in the text definition itself, just in the dbxref. Would you mind revising that? Everything else looks good, thanks!

**Review by nicolevasilevsky — CHANGES_REQUESTED** [review]
one more minor change needed, thanks!

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo:455495]: @nicolevasilevsky I can remove them, though I am questioning whether this whole part of the "definition" release belongs to the definition... Maybe it should be a comment about this term?

Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:455495]: I agree- that seems appropriate for a comment. Thanks!

Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo:455495]: Thank you! I updated the definition and comment

**Review by nicolevasilevsky — APPROVED** [review]

**Pull request merged**